examples/pytorch/gin/README.md

```diff
@@ -4,80 +4,23 @@ Graph Isomorphism Network (GIN)
 - Paper link: [arXiv](https://arxiv.org/abs/1810.00826) [OpenReview](https://openreview.net/forum?id=ryGs6iA5Km) 
 - Author's code repo: [https://github.com/weihua916/powerful-gnns](https://github.com/weihua916/powerful-gnns).
 
-Dependencies
-------------
-- PyTorch 1.1.0+
-- sklearn
-- tqdm
-
-``bash
-pip install torch sklearn tqdm
-``
```

Comment:
I think `sklearn` is still a dependency.

Reply:
Thanks. I updated.